### PR TITLE
Add admin lock command

### LINKED ISSUE #1920: admin: Implement lock command
## Expected behaviour
Having a lock command to list/clear locks hold by a specified minio server

## Actual behaviour
Not having lock command

## mc version
All
## System information
All

### PR DESCRIPTION
**Pull request by vadmeste**
Fixes #1920

### CODE REVIEW

**Comment by harshavardhana** [comment]
Lock has been merged on server , please rebase @vadmeste

**Comment by vadmeste** [comment]
Ready for review

File: cmd/admin-lock-clear.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+USAGE:
+   mc admin lock {{.Name}} ALIAS/BUCKET/PREFIX
+
+FLAGS:
+  {{range .Flags}}{{.}}
+  {{end}}
+EXAMPLES:
+    1. Clear all locks hold in a Minio server represented by its alias 'play'.
+       $ mc admin lock {{.Name}} play/
```
Comment (harshavardhana) [inline cmd/admin-lock-clear.go]: Can you add more examples?

File: cmd/admin-lock-clear.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+	isSSL := (url.Scheme == "https")
+
+	// Create a new Minio Admin Client
+	client, e := madmin.New(url.Host, hostCfg.AccessKey, hostCfg.SecretKey, isSSL)
+	if e != nil {
+		return e
+	}
+
+	aliasedURL = strings.Replace(aliasedURL, "\\", "/", -1)
```
Comment (harshavardhana) [inline cmd/admin-lock-clear.go]: use `filepath.ToSlash` and `filepath.FromSlash` helpers.

File: cmd/admin-lock-clear.go (hunk 1 above)
Comment (harshavardhana) [inline cmd/admin-lock-clear.go]: Atleast 3.

File: cmd/admin-lock-clear.go [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+USAGE:
+   mc admin lock {{.Name}} ALIAS/BUCKET/PREFIX
+
+FLAGS:
+  {{range .Flags}}{{.}}
+  {{end}}
+
+EXAMPLES:
+    1. Clear all locks hold by 'testbucket' in a Minio server represented by its alias 'play'.
+       $ mc admin lock {{.Name}} play/testbucket
```
Comment (harshavardhana) [inline cmd/admin-lock-clear.go]: Add a space between each examples.

File: cmd/admin-lock-list.go [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+USAGE:
+   mc admin lock {{.Name}} ALIAS/BUCKET/PREFIX
+
+FLAGS:
+  {{range .Flags}}{{.}}
+  {{end}}
+
+EXAMPLES:
+    1. List hold locks related to testbucket in a Minio server represented by its alias 'play'.
```
Comment (harshavardhana) [inline cmd/admin-lock-list.go]: Same as above.

**Pull request merged**